packages/run-protocol/src/vaultFactory/prioritizedVaults.js

```diff
@@ -1,4 +1,5 @@
 // @ts-check
+// @jessie-check
```

Comment:
This is more strict that it needs to be. https://github.com/Agoric/agoric-sdk/issues/4770

Reply:
but it passes so I thought I'd leave it.